nvtabular/graph/graph.py

```diff
@@ -80,6 +57,11 @@ def fit_schema(self, root_schema: Schema) -> "Graph":
 
         return self
 
+    def update_properties(self):
+        for node in postorder_iter_nodes(self.output_node):
+            if node.parents:
+                node.op.update_properties()
```

Comment:
I don't think `update_properties` still exists, so we should replace this with the newer version.